packages/venia-ui/lib/components/SearchPage/searchPage.js

```diff
@@ -103,6 +104,10 @@ const SearchPage = props => {
                 </div>
                 {maybeSortContainer}
             </div>
+            <div className={classes.heading}>
+                {'Showing results for '}
+                <span className={classes.headingHighlight}>{searchTerm}</span>:
```

Comment:
Taking a value out of the URL and displaying it on the page would normally open us up to XSS but [React / JSX takes care of this for us](https://reactjs.org/docs/introducing-jsx.html#jsx-prevents-injection-attacks).